### DISCUSSION TITLE
Migrating Markers and photos from 3.0.4

### DISCUSSION THREAD

**Discussion by johnea** [osmandapp/OsmAnd#16455] [Q&A]
Hello,

I've been using the same old 3.0.4 version of OsmAnd for years.

Now, I'm moving to a new phone and I'd like to migrate my markers and photos into a new OsmAnd installation (on android).

My current phone is rooted, and I have access to adb and other tools.

How can I copy my existing Markers and photos from 3.0.4, and install them in a fresh OsmAnd version?

Thank You!
johnea
2023-02-13

**Comment by pebogufi** [comment]
There are a lot of problems using Android 11 and newer, if this concerns you, search for the threads.
- Photos and audio notes. If you think about these notes just copy the folder avnotes to the new device.
- markers. You can save the markers as track file and import in other device. Just have look in the options of markers.

**Reply by johnea** [reply]
Thank you pebogufi! 

I did find the export as GFX file. Flashing new phone now, will test import once complete

Also found the avnotes directory. Will copy that as well.

Thank you for the specific directory name and instructions!

I did search around re Android 11, and found the issue of SDCard storage access changing.

It seemed like this had been resolved though. Is this not so? Or are there other issues with Android newer than 11?

Thank you again for your guidance...

johnea
2023-02-17

**Reply by johnea** [reply]
I'm only able to import the GPX file as "My Favorites" 8-(

Is there a way to get Markers from an old osmand install moved into a new install?

The phone is rooted, and I have adb. I don't mind going through some gyrations if it can restore teh points as Markers.

I like the UI of notating them, and the ability to move them if I want to update them.

Thank you for your guidance...

**Comment by johnea** [comment]
Maybe a better question is just: What file contains the markers in an osmand install?

**Reply by johnea** [reply]
OK, I found the file:
data/data/net.osmand/databases/map_markers_db

Which exists on the old 3.0.4 version, with all the markers I want to migrate.

Also exists on the new 4.3.12 install. Although I created a test marker in 4.3.12, and I see that another file is modified:
/data/media/0/Android/data/net.osmand/files/itinerary.gpx

Can I just Force Stop osmand, and copy in this database? Is the schema the same?

Thank you again for any guidance...

**Reply by johnea** [reply]
OK, the short answer seems to be Yes.

Copying in the map_markers_db from the old install to overwrite the file in the new install at:
 data/data/net.osmand/databases/map_markers_db

Does show all of the markers when osmand is launched.

Not sure if there will be consequences, if anyone is aware of issues with such a direct copy, please post here!

Thank You!!
